ast/src/generic.rs

```diff
@@ -45,4 +54,251 @@ impl<R> Default for EmptyRange<R> {
     }
 }
 
+impl<R> Arguments<R> {
+    pub fn empty(range: OptionalRange<R>) -> Self {
+        Self {
+            range,
+            posonlyargs: Vec::new(),
+            args: Vec::new(),
+            vararg: None,
+            kwonlyargs: Vec::new(),
+            kwarg: None,
+        }
+    }
+}
+
+#[allow(clippy::borrowed_box)] // local utility
+fn clone_boxed_expr<R: Clone>(expr: &Box<Expr<R>>) -> Box<Expr<R>> {
+    let expr: &Expr<_> = expr.as_ref();
+    Box::new(expr.clone())
+}
+
+impl<R> ArgWithDefault<R> {
+    pub fn from_arg(def: Arg<R>, default: Option<Expr<R>>) -> Self
+    where
+        R: Clone,
+    {
+        #[allow(clippy::useless_conversion)] // false positive due to cfg
+        let range = OptionalRange::from(def.range.clone()); // FIXME: def.range.start()..default.range.end()
```

Comment:
I think this `Fixme` is still relevant. Do you intend to fix it before merging?

Reply:
I intended to leave it for this patch. But it makes me think this is too naive way to leave a bug. It can be a todo! panic for `feature = "more-attributes"` for now. No user hits this bug yet.